domain-cc/cc-app/src/python_src/api.py

```diff
@@ -10,6 +10,17 @@
 LOOKUP_TABLE = get_lookup_table()
 
 app = FastAPI(
+    title="Contention Classification",
+    description="Mapping contention text to actual classifications per `/contention-classification/classifier` defined for [Benefits Reference Data API](https://developer.va.gov/explore/benefits/docs/benefits_reference_data).",
+    contact={
+        "name": "Premal Shah",
+        "email": "premal.shah@va.gov"
+    },
+    version="v0.2",
+    license={
+        "name": "CCO 1.0",
```

Comment:
If this license info is stored in VRO, it would be better to just verbatim include that, but it's unlikely to change.

It would also make sense to move these settings out, but I can do that in a separate PR.